src/utils/api/helper.ts

```diff
@@ -328,18 +332,40 @@ export const getPaymentPSPList = async ({
     O.getOrElseW(() => undefined)
   );
 
-  const lang = "it";
+  const transferList: Array<TransferListItem> = pipe(
```

Comment:
Wouldn’t it be better to save the **trasferList** after the activation ?
 -----> **handler.js** | method **activatePayment**

Reply:
Since calculateFee is used also in pspEditClick listener this variable has to be evaluated in this point of the code.